models/org.go

```diff
@@ -749,23 +749,36 @@ func (org *User) AccessibleReposEnv(userID int64) (AccessibleReposEnvironment, e
 }
 
 func (org *User) accessibleReposEnv(e Engine, userID int64) (AccessibleReposEnvironment, error) {
+	var user *User
+
+	if userID > 0 {
+		u, err := getUserByID(e, userID)
```

Comment:
```suggestion
		user, err := getUserByID(e, userID)
```

Reply:
This will define new internally scoped user variable

Reply:
Ok